pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.black]
+line-length = 100
```

Comment:
How do you feel about 120 :)

Reply:
No strong opinions here. As a test, I tried converting a bunch of my personal repositories to Black, and I tried 88, 100, and 120. In my opinion, 100 looked the most readable.